src/template.html

```diff
@@ -426,6 +426,11 @@ <h3>{{topic.others.by-us-title}}</h3>
           summaryToggleButton.setAttribute('aria-expanded', 'false');
         }
 
+        setTimeout(() => {
+          themeToggleButton.classList.remove("hidden");
+          themeToggleButton.classList.add("transition");
+        }, 0);
+
```

Comment:
Is it an option to not do this with JS but just make it the default in the html? I tried locally to replace this JS with `class="theme-toggle transition"` on the `input` for the theme switcher and it worked just as well, as far as I could tell?

Reply:
We discussed this offline and I'll insert another suggestion.